catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -258,6 +260,16 @@ public ResourceResponse retrieveResource(URI resourceURI, Map<String, Serializab
     if (resourceURI.getScheme().equals(URL_HTTP_SCHEME)
         || resourceURI.getScheme().equals(URL_HTTPS_SCHEME)) {
       LOGGER.debug("Resource URI is HTTP or HTTPS");
+
+      final Serializable qualifierSerializable = properties.get(ContentItem.QUALIFIER);
+      if (qualifierSerializable instanceof String) {
+        final String qualifier = (String) qualifierSerializable;
+        if (StringUtils.isNotBlank(qualifier)) {
+          resourceURI =
+              UriBuilder.fromUri(resourceURI).queryParam(ContentItem.QUALIFIER, qualifier).build();
+        }
+      }
```

Comment:
This addition is copied from the corresponding method in ContentResourceReader:
https://github.com/codice/ddf/blob/3efc289a995afbddf1170c716233a34ee82eaab8/catalog/core/catalog-core-contentresourcereader/src/main/java/org/codice/ddf/catalog/content/resource/reader/ContentResourceReader.java#L159-L171